src/csrc/rl/off_policy/interface.cpp

```diff
@@ -198,16 +202,16 @@ torchfort_result_t torchfort_rl_off_policy_update_replay_buffer(const char* name
     switch (dtype) {
     case TORCHFORT_FLOAT: {
       float reward_val = *reinterpret_cast<const float*>(reward);
-      rl::update_replay_buffer<RowMajor>(name, reinterpret_cast<float*>(state_old), reinterpret_cast<float*>(state_new),
-                                         state_dim, state_shape, reinterpret_cast<float*>(action_old), action_dim,
-                                         action_shape, reward_val, final_state, ext_stream);
+      rl::off_policy::update_replay_buffer<RowMajor>(name, reinterpret_cast<float*>(state_old), reinterpret_cast<float*>(state_new),
+						     state_dim, state_shape, reinterpret_cast<float*>(action_old), action_dim,
```

Comment:
Nit. Whitespace (here and in other parts of this file.).

Reply:
I went over the file and reformatted some lines, especially I broke some of the very long lines and replaced some tabs with whitespaces to have everything consistent. Please have a look.

Reply:
we fix the formatting after the merge globally